src/ensembl/src/services/storage-service.ts

```diff
@@ -44,10 +50,20 @@ const defaultOptions: options = {
 };
 
 // We need to overwrite the arrays instead of merging them so that it is easier to remove entries
-const overwriteArray = (currentValue: JSONValue, newValue: JSONValue) => {
+const overwriteArray = (
```

Comment:
The name of the function no longer properly reflects what it does. Perhaps rename the function to something like mergeStrategy?

Reply:
Changed it to `mergeStrategy`